packages/babel-plugin/src/class-names/index.tsx

```diff
@@ -26,14 +63,37 @@ const extractStyles = (path: NodePath<t.Expression>): t.Expression[] | t.Express
 
   if (
     t.isCallExpression(path.node) &&
-    t.isMemberExpression(path.node.callee) &&
-    t.isIdentifier(path.node.callee.property) &&
-    path.node.callee.property.name === 'css' &&
-    t.isExpression(path.node.arguments[0])
+    t.isIdentifier(path.node.callee) &&
+    t.isExpression(path.node.arguments[0]) &&
+    path.scope.hasOwnBinding(path.node.callee.name)
   ) {
-    // props.css({}) call
-    const styles = path.node.arguments as t.Expression[];
-    return styles;
+    const binding = path.scope.getBinding(path.node.callee.name)?.path.node;
+
+    if (isTypeComingFromDestructuring('css', binding as t.Expression)) {
+      // c({}) rename call
+      const styles = path.node.arguments as t.Expression[];
+      return styles;
+    }
+  }
+
+  if (t.isCallExpression(path.node) && t.isMemberExpression(path.node.callee)) {
+    // filter out invalid calls like dontexist.css({})
+    if (
+      t.isIdentifier(path.node.callee.object) &&
+      !path.scope.hasOwnBinding(path.node.callee.object.name)
```

Comment:
do we need to handle `doesexist.css`? (except its just another variable from somewhere else)

Reply:
let me check once if its not needed

Reply:
How about this case? It will break if we don't handle this case. If `css` is coming from outside of scope.

```ts
import { ClassNames } from '@compiled/react';

const ListItem = () => (
  <ClassNames>
    {(props) => (<div style={dontexist.style} className={dontexist.css({ fontSize: '20px' })}>hello, world!</div>)}
  </ClassNames>
);
```

Reply:
arent we already handling this 🤔 or ur saying dontexist.css should be shouldexist.css?

i reckon let's not add too much if we don't need it. this is pretty hypothetical atm 😄